---
pr2_guide/SpotArray ret
